src/Amazon.Extensions.CognitoAuthentication/Util/AuthenticationHelper.cs

```diff
@@ -208,6 +209,7 @@ public static DeviceSecretVerifierConfigType GenerateDeviceVerifier(string devic
         /// <param name="tupleAa"> TupleAa from CreateAaTuple</param>
         /// <returns>Returns the claim for authenticating the given user</returns>
         public static byte[] AuthenticateDevice(
+            string username,

```

Comment:
Adding a param to a public method is a breaking change in both this case and the one below.